std/http/headers.zig

```diff
@@ -102,9 +102,19 @@ test "HeaderEntry" {
     testing.expectEqualSlices(u8, "x", e.value);
 }
 
+fn stringEql(a: []const u8, b: []const u8) bool {
+    if (a.len != b.len) return false;
+    if (a.ptr == b.ptr) return true;
+    return mem.compare(u8, a, b) == .Equal;
+}
+
+fn stringHash(s: []const u8) u32 {
+    return @truncate(u32, std.hash.Wyhash.hash(0, s));
```

Comment:
Wouldn't it be more desirable (for DoS avoidance) to have a unique seed for each `Headers` struct, instead of using a single hardcoded seed value for all `Headers` structs?